src/config.h

```diff
@@ -17,10 +17,12 @@
 #include <sys/types.h>
 // typedef uni32_t
 #include <stdint.h>
-// assert_sizeof
-#include "static_assert.h"
 // struct in_addr, in6_addr
 #include <netinet/in.h>
+// type bool
+#include <stdbool.h>
+// type FILE
+#include <stdio.h>
```

Comment:
Why are those two `include` needed now?

Reply:
We remove the `assert.h` dependency which causes some automatic "behind-the-scenes" includes to not happen. Being more explicit what we want seems a better idea in general.